dist/split-test.js

```diff
@@ -17,6 +17,8 @@
     }
 }(this, function() {
 
+    var DEFAULT_LIFETIME = 30 * 24 * 3600;
```

Comment:
This variable is also used in the SplitTest constructor. So if we want to change it, we need to make sure that all time variable is in the same unit (ie. all in seconds or milliseconds)